google/cloud/bigquery/client.py

```diff
@@ -2757,7 +2760,14 @@ def load_table_from_json(
 
         job_config.source_format = job.SourceFormat.NEWLINE_DELIMITED_JSON
 
-        if job_config.schema is None:
+        # make table id
+        # table_id = "your-project.your_dataset.your_table"
+        destination = _table_arg_to_table_ref(destination, default_project=self.project)
```

Comment:
Why are we calling `_table_arg_to_table_ref()` here?
The very first line of `get_table()` calls that function.

table_ref = _table_arg_to_table_ref(table, default_project=self.project)